server/models/user.rb

```diff
@@ -33,4 +33,9 @@ def password_matches?(raw_password)
   def self.public_attrs
     [:username, :id]
   end
+  def avatar_url
+    hash = Digest::MD5.hexdigest(if self.email then self.email.strip.downcase else "" end)
+    self.avatar ? "/api/auth/users/#{self.id}/avatar"
```

Comment:
The only downside about defining the path in the user model is that it may break if we ever change where the controller is mounted in the rack application (or the paths).  I know that rails uses `link_to` helpers; we may want to investigate something similar for this app.